Source/ablastr/particles/NodalFieldGather.H

```diff
@@ -12,16 +12,19 @@
 #include <AMReX_Array.H>
 #include <AMReX_Extension.H>
 #include <AMReX_GpuQualifiers.H>
+#include <AMReX_IndexType.H>
 #include <AMReX_Math.H>
 #include <AMReX_REAL.H>
 
+
 namespace ablastr::particles
 {
+
 /**
  * \brief Compute weight of each surrounding node (or cell-centered nodes) in interpolating a nodal ((or a cell-centered node) field
- *        to the given coordinates. If nodal=1, then the calculations will be done with respect to the nodes (default). If nodal=0, then the calculations will be done with respect to the cell-centered nodal)
- *
- * This currently only does linear order.
+ *        to the given coordinates. If template parameter IdxType is amrex::IndexType::CellIndex::NODE, then the calculations will be done with respect to the nodes (default).
```

Comment:
Is the documented default set?
```suggestion
 *        to the given coordinates. If template parameter IdxType is amrex::IndexType::CellIndex::NODE, then the calculations will be done with respect to the nodes.
```